config.yaml

```diff
@@ -39,10 +39,10 @@
             # Make sure that the samples you merge have the same sample features (e.g paired, sense, dup_type)
             # Choose an informative experiment name
   experiment_name: {
-                    rbp: 'rbp name', #rna binding protein name, eg: PUM2,
```

Comment:
In this section there are some fields that the user can choose, e.g. I presume "rpb_name", and others that will be predetermined (e.g. file with sequences). This should be made clear.